examples/advancedsnippet.txt

```diff
@@ -46,3 +47,9 @@ Compare xyz to 1, show 'xyz is not 1' if false:
 	if(true) block
 {{ } }}
 
+Create a series of elements using a loop at runtime
+<ul>
```

Comment:
Object iteration in templates is usually not a good idea. See https://github.com/olado/doT/pull/124#issuecomment-257169745